tee-worker/service/src/config.rs

```diff
@@ -61,7 +62,8 @@ pub struct Config {
 	pub running_mode: String,
 
 	/// Litentry
-	pub disable_mock_server: bool,
+	pub mock_server: bool,
```

Comment:
Maybe 'mock_server_enable' or something similar would be more intuitive.

Reply:
had adjusted.

Reply:
It would be better to follow the document format to add some description as comment. Because it will appear at the command line help menu.